test/serializers/attribute_test.rb

```diff
@@ -81,6 +81,23 @@ def id
         assert_equal('custom', hash[:blog][:id])
       end
 
+      def test_id_with_question_mark_has_question_mark_removed
+        serializer = Class.new(ActiveModel::Serializer) do
+          # TODO: if we were to remove the question mark automatically,
+          #       what do we do if we want to call a method that has a
+          #       question mark.
+          #       e.g.:
+          #               def my_question?; ....; end
```

Comment:
yeah, but I mean if someone wanted to use a method in their model rather than just a db-backed field.

e.g.:
```ruby
2.2.3 :001 > def test?; 'hi'; end
 => :test? 
2.2.3 :002 > test
ArgumentError: wrong number of arguments (0 for 2..3)
	from (irb):2:in `test'
	from (irb):2
	from /home/preston/.rvm/rubies/ruby-2.2.3/bin/irb:11:in `<main>'
2.2.3 :003 > test?
 => "hi" 
2.2.3 :004 > 
```